src/Domain/Event/ValueObject/EventFilter.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Domain\Event\ValueObject;
+
+class EventFilter
```

Comment:
I think this class is not needed. The `App\Application\Command\ReplayEventsCommand` is responsible of parse the input in a way that domain can understand.